### PR TITLE
Created To Do List basic structure

### PR DESCRIPTION
**Pull request by sebalp100**
### **Created To Do List basic structure**

For this milestone we added:

- A basic HTML file to create new tasks elements
- Basic styling with CSS following the template
- Used JavaScript array of Objects to create the new tasks and display them in the browser
- Set up initial local storage in preparation for the next project
- Used webpack to bundle JavaScript.
- Added instructions on how to use webpack to deploy this project in the README file.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    if (!data) {
+      data = [];
+    }
+    input.value = '';
+    const taskInfo = { description: task, completed: false, index: data.length };
+    data.push(taskInfo);
+    localStorage.setItem('todo-list', JSON.stringify(data));
+    Actions.showTask(data);
+  }
+});
```
Comment (codepantha) [inline src/index.js]: - [x] For my milestone, you're not required to implement the add to-do functionality like you have done here. Please simply create an array of to-do tasks and iterate over them. See an example of what the array of to-do objects might look like 👇 
```javascript
const tasks = [
  {
    description: 'An example description in string',
    completed: false,
    index: 10,
  },
];
```

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: REQUIRES CHANGES ♻️ 

## Hi @sebalp100 👋, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!


## Highlights 🎯 
✔️ Descriptive PR
✔️ Used GitHub Flow
✔️ No linter errors

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (sebalp100) [reply src/index.js]: Hi, yeah sorry. I was trying to set it up for the next project. Should I just create a hardcoded array for this occasion?

**Comment by sebalp100** [comment]
Changes required implemented. Added a hard-coded array with example tasks.

**Review by ridwanediallo — APPROVED** [review]
# STATUS: APPROVED 🟢
## Hi @sebalp100 , 
Congratulations!GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- GitHub flow is used correctly. ✔
- All linters checks are ok. ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**